app/screens/select_server/index.js

```diff
@@ -22,9 +22,10 @@ function mapStateToProps(state) {
     const {currentVersion, latestVersion, minVersion} = getClientUpgrade(state);
 
     return {
-        ...state.views.selectServer,
+        serverURL: getCurrentUrl(state),
```

Comment:
The prop being passed here is `serverURL`, but the expected prop in the child component is `serverUrl`, which is likely always initially being set to `undefined` now.

Reply:
good catch @chuttam thanks, I am fixing that